### ISSUE TITLE
Wrong recognition of the bone location in AudioAnimationTools

### ISSUE THREAD

**Issue by Chan-Johann** [DemuteStudio/UnrealAudioTools#8]
Issue found on UE 5.4 and seen before on 5.3

After loading up the widget tool and setting it up to detect movement of the character no events are being created and log presents error "Didn't create any footstep on track AutoGen foot_l of animation sequence Enemy_CautiousRun. The ground contact values might be wrong. The lowest position for this bone in the animation was 0 and the highest position was 0."

same thing happend in our project and on the vanila default Unreal set of animations - plugin is not detecting current world location of bones in animation and keeps returning values that do not change and therefore not creating any events on timeline, even if we modify the position of bone foot_l to ridiculus heights
![obraz_2024-11-21_140041069](https://github.com/user-attachments/assets/58b75739-e507-453e-8ead-9b654b80fe59)
![obraz_2024-11-21_140113796](https://github.com/user-attachments/assets/af6d172e-1f66-459c-b3f5-457c66e54782)

**Comment by DemuteStudio** [comment]
Hey! Thanks for your post.

Which version of the plugin are you using? The latest release (1.2.0)? Or are you compiling from source?

Unreal 5.3 introduced breaking changes that we have corrected on the branch version/unreal-5.3 but we haven't made a package from it yet. Happy to do so if it helps you tho!

**Comment by Chan-Johann** [comment]
Thanks! that's what I needed, we used the Main 1.2.0 version and did not expect that other branch will be actually for newer UE version :)

I now checked branch version/unreal-5.3 on fresh project and it worked properly, so will be integrating it with our project and Wwise on monday). 

Cool to finally find that tool working! might be good to do add some mention about bugs and other versions somewhere on git for others, thanks again for help and cool tool!
